pkg/clusteragent/admission/mutate/config/config.go

```diff
@@ -217,6 +232,53 @@ func injectionMode(pod *corev1.Pod, globalMode string) string {
 	return globalMode
 }
 
+// injectExternalDataEnvVar injects the External Data environment variable.
+// The format is: it-<init>,cn-<container_name>,pu-<pod_uid>
+func injectExternalDataEnvVar(pod *corev1.Pod) bool {
```

Comment:
nit: you can use named returns in golang to make it more obvious what the returned boolean signifies

Reply:
Done 👍 I've not used a naked return though to maintain readability.